baseplate/sidecars/live_data_watcher.py

```diff
@@ -3,6 +3,7 @@
 import configparser
 import logging
 import os
+import pathlib
```

Comment:
might be nice to `from pathlib import Path` like you do in the tests. would look a bit nicer below.